### PR TITLE
Add inspector build script

### PR DESCRIPTION
**Pull request by jasssonpet**
@KristinaKoeva, could you give a hand with the code signing?

This should add no more than a minute to the build time (except the first time).

### CODE REVIEW

File: build/scripts/build-inspector.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    SYMROOT="$INSPECTOR_BUILD_OUTPUT_PATH" \
+    build \
+    >> "$WORKSPACE/build.log" 2>&1
+
+echo "Packaging Inspector app..."
+pushd $INSPECTOR_BUILD_OUTPUT_PATH/$CONFIGURATION/ \
+    >> "$WORKSPACE/build.log" 2>&1
+rm -f "NativeScript Inspector.zip"
+rm -rf "NativeScript Inspector.app" "NativeScript Inspector.app.dSYM"
+mv "Inspector.app" "NativeScript Inspector.app"
```
Comment (jasssonpet) [inline build/scripts/build-inspector.sh]: Can someone verify these lines here. The new Inspector archive seems to have the frameworks duplicated.

File: build/scripts/build-inspector.sh [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+mkdir -p "$WORKSPACE/cmake-build"
+
+WEBKIT_SOURCE_PATH="$WORKSPACE/src/webkit"
+WEBKIT_BUILD_OUTPUT_PATH="$WORKSPACE/cmake-build/WebKit-Xcode"
+
+INSPECTOR_SOURCE_PATH="$WORKSPACE/src/debugging/Inspector/Inspector"
+INSPECTOR_BUILD_OUTPUT_PATH="$WORKSPACE/cmake-build/Inspector"
+
+echo "Building WebKit..."
+xcodebuild \
```
Comment (fealebenpae) [inline build/scripts/build-inspector.sh]: Use the `Tools/Scripts/build-webkit` script rather than manually building.

Comment (jasssonpet) [inline build/scripts/build-inspector.sh]: Oh, nice. It even has a `--inspector-frontend` flag that we can use.

Comment (jasssonpet) [inline build/scripts/build-inspector.sh]: @fealebenpae I'm experiencing some failing builds using the `build-webkit` tool. Do you have any idea of: `fatal error: 'wtf/Platform.h' file not found'`?

File: build/scripts/build-inspector.sh (hunk 1 above)
Comment (jasssonpet) [inline build/scripts/build-inspector.sh]: Building with `archive` instead of `build` seems to do the work.

**Comment by jasssonpet** [comment]
I believe this PR is complete for now.

We only have to configure the certificate on our build machines.

File: src/debugging/Inspector/Inspector/Inspector.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -331,7 +331,6 @@
 		51E6FCC91B42E25500FF4242 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
-				CODE_SIGN_IDENTITY = "Developer ID Application";
```
Comment (jasssonpet) [inline src/debugging/Inspector/Inspector/Inspector.xcodeproj/project.pbxproj]: I've moved the code signing settings to project level.

**Comment by jasssonpet** [comment]
run ci

**Comment by jasssonpet** [comment]
run ci

**Comment by jasssonpet** [comment]
run ci

**Comment by jasssonpet** [comment]
run ci

**Comment by KristinaKoeva** [comment]
run ci

**Comment by jasssonpet** [comment]
run ci

**Comment by jasssonpet** [comment]
run ci

**Comment by jasssonpet** [comment]
run ci

**Pull request merged**